packages/js/product-editor/src/hooks/use-product-variations-helper.ts

```diff
@@ -158,19 +158,6 @@ export function useProductVariationsHelper() {
 				}
 			)
 			.then( async ( response ) => {
-				// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-				// @ts-ignore
-				await dispatch( 'core' ).invalidateResolution(
-					'getEntityRecord',
-					[ 'postType', 'product', productId ]
-				);
-
-				await resolveSelect( 'core' ).getEntityRecord(
-					'postType',
-					'product',
-					productId
-				);
-
```

Comment:
Don't need to trigger the above anymore as `refreshProductVariationData` is triggered within the `generateVariations` action.